### DISCUSSION TITLE
请问下是否支持只使用量算和计量功能，是否有暴露对应api

### DISCUSSION THREAD

**Discussion by Sfhji** [zouyaoji/vue-cesium#528] [Q&A]
使用vue-cesium，引入cesium1.112版本，开启move事件，会产生掉帧。
但是如果只使用原生的cesium 1.112版本不会产生掉帧。

所以想问下是否支持只使用量算和计量功能，是否有暴露对应api

谢谢大佬。

**Comment by zouyaoji** [comment]
掉帧的问题建议提供在线复现代码。

有个计算面积的方法 https://github.com/zouyaoji/vue-cesium/blob/dev/packages/utils/cesium-helpers.ts#L966

**Reply by Sfhji** [reply]
谢谢大佬回复。掉帧问题，因为把viewer挂到了vue上，导致了严重掉帧。麻烦了。
